src/protocol/generate-python.py

```diff
@@ -64,6 +64,7 @@ def is_vector(payload):
 allMessages.update(messageDict['general'])
 
 json_data['all_msgs'] = allMessages
+json_data['all_fields'] = list(set([msg['name'] for key, message in allMessages.items() for msg in message['payload']]))
```

Comment:
This is a pretty complex statement. Please explain, or break it down for readability.

Reply:
oops, I was going to add a comment explaining but forgot, my bad.